cmd/influxd/run.go

```diff
@@ -71,6 +72,16 @@ func execRun(args []string) {
 		}
 		log.Printf("data node #%d listening on %s", s.ID(), config.DataAddr())
 
+		// Spin up the collectd server
+		if config.Collectd.Enabled {
+			c := config.Collectd
+			cs := collectd.NewServer(s, c.TypesDB)
+			cs.Database = c.Database
+			err := collectd.ListenAndServe(cs, c.ConnectionString(config.BindAddress))
```

Comment:
By the way I am torn how much we want to change the API pattern we have been following, since this is code purely for internal use only. But I wanted to check with @davecheney.

Let me know what you think too, @corylanou.